docs/docs/homepage/homepage.md

```diff
@@ -0,0 +1,48 @@
+# 👋 Welcome to BastionLab!
```

Comment:
Is this a duplicate of the index.md file?

Reply:
I think so. This was in the original docs branch. If it's not useful, we can just remove it.

Reply:
I was wondering why that was there actually because it's just a duplicate yeah.